lib/data/readme.md

```diff
@@ -0,0 +1,26 @@
+# Introduction
+
+The `lib/data` folder has all our crowdsourced data files.
+This readme explains what each file is used for.
+
+## Summary
+
+| File            | What is the file about?                  |
+| --------------- | ---------------------------------------- |
+| `monorepo.json` | Group related packages into a single PR. |
+
+## Group related packages (`monorepo.json`)
+
+The `monorepo.json` file has all the monorepo presets.
+
+Monorepo presets group related packages, so they are updated with a single Renovate PR.
+
+### Ways to group packages
+
+There are three ways to group packages:
+
+| Method          | Group packages                      |
+| --------------- | ----------------------------------- |
+| `repoGroups`    | From the same source repository.    |
+| `orgGroups`     | From the same organization.         |
+| `patternGroups` | Based on name patterns or prefixes. |
```

Comment:
Great idea to put everything into two rows, and removing the duplication! :smile: 

We should put the simple info up-front, like this:

```suggestion
| Group packages                      | Method          |
| ----------------------------------- | --------------- |
| From the same source repository.    | `repoGroups`    |
| From the same organization.         | `orgGroups`     |
| Based on name patterns or prefixes. | `patternGroups` |
```